mealpy.py

```diff
@@ -121,14 +128,72 @@ def cancel_current_meal(self):
         raise NotImplementedError()
 
 
-SCHEDULER = BlockingScheduler()
+def get_mealpal_credentials():
+    email = CONFIG['email_address']
+    if CONFIG['use_keyring']:
+        password = (
+            keyring.get_password(KEYRING_SERVICENAME, email)
+            or getpass.getpass('Credential not yet stored in keychain, '
+                               'please enter password: ')
```

Comment:
Might I throw in a code styling suggestion:
```python
or getpass.getpass(
    'Credential not yet stored in keychain, ' 'lots more text that eventually exceeds max-line-length'
    'please enter password: ')
)
```

Reply:
Actually going to leave it as 
```python
or getpass.getpass('Credential not yet stored in keychain, please enter password: ')
```
since it wasn't exceeding line length before and the newline I added was not necessary. 
If we add more text in the future we can implement your suggestion.